Dockerfile

```diff
@@ -2,17 +2,28 @@ FROM centos:7
 MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
 
 RUN yum update -y && \
-    yum install -y golang openscap-scanner && \
+    yum install -y \
+    openscap-scanner && \
     yum clean all
 
+    #needs go 1.7 for context
+RUN cd /tmp && \
+    curl -LO https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz && \
+    tar -C /usr/local -xvzf go1.7.linux-amd64.tar.gz && \
+    rm -f go1.7.linux-amd64.tar.gz
```

Comment:
true. this will be resolved anyway once we merge https://github.com/openshift/image-inspector/pull/59 this will be irrelevant; however until then i'm make sure we remove `go` once the binary is built